Calculator/Calculator/Model/Operation.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Operator.swift
+//  Calculator
+//
+//  Created by KimChoonSik on 2023/01/17.
+//
+
+import Foundation
+
+
+enum Operation: String, CalculateItem {
+
+    case add = "+"
+    case subtract = "-"
+    case divide = "÷"
+    case multiply = "×"
+    case calculate = "="
+
+    func add(previousData: Double, nextData: Double) -> Double {
+        return previousData + nextData
+    }
+    func substract(previousData: Double, nextData: Double) -> Double {
+        return previousData - nextData
+    }
+    func divide(previousData: Double, nextData: Double) -> Double {
+        return previousData / nextData
+    }
+    func mulitiply(previousData: Double, nextData: Double) -> Double {
+        return previousData * nextData
+    }
+    func calculate(previousData: Double, nextData: Double) -> Double {
+        return previousData + nextData
+    }
```

Comment:
```suggestion
    func add(previousData: Double, nextData: Double) -> Double {
        return previousData + nextData
    }
    
    func substract(previousData: Double, nextData: Double) -> Double {
        return previousData - nextData
    }
    
    func divide(previousData: Double, nextData: Double) -> Double {
        return previousData / nextData
    }
    
    func mulitiply(previousData: Double, nextData: Double) -> Double {
        return previousData * nextData
    }
    
    func calculate(previousData: Double, nextData: Double) -> Double {
        return previousData + nextData
    }
```
=> 메소드간에 띄워쓰기 부탁드립니다 🙏

Reply:
코드 가독성을 위해 메소드간 띄워쓰기 신경쓰겠습니다 :)